website/src/data/buttons.json

```diff
@@ -851,5 +851,23 @@
                 "usesColors": false
             }
         ]
+    },
+    {
+        "type": "multistate",
+        "label": "Multistate Buttons",
+        "classes": "multi-state-btn",
+        "data": [

```

Comment:
What's this `data` key for? Please make sure you follow the structure detailed in the contributing guidelines.

Reply:
The button will need the data-states="a number" attribute to work.

Reply:
Should I remove the `data` key?